apps/condo/migrations/20230927122341-0322_auto_20230927_0725.js

```diff
@@ -0,0 +1,121 @@
+
+exports.up = async (knex) => {
+    await knex.raw(`
+BEGIN;
+--
+-- [CUSTOM] Set Statement Timeout to some large amount - 25 min (25 * 60 => 1500 sec)
+--
+SET statement_timeout = '1500s';
+
+--
+-- Add field canReadContacts to organizationemployeerole
+--
+ALTER TABLE "OrganizationEmployeeRole" ADD COLUMN "canReadContacts" boolean DEFAULT true NOT NULL;
+ALTER TABLE "OrganizationEmployeeRole" ALTER COLUMN "canReadContacts" DROP DEFAULT;
+--
+-- Add field canReadEmployeeRoles to organizationemployeerole
+--
+ALTER TABLE "OrganizationEmployeeRole" ADD COLUMN "canReadEmployeeRoles" boolean DEFAULT true NOT NULL;
+ALTER TABLE "OrganizationEmployeeRole" ALTER COLUMN "canReadEmployeeRoles" DROP DEFAULT;
+--
+-- Add field canReadEmployees to organizationemployeerole
+--
+ALTER TABLE "OrganizationEmployeeRole" ADD COLUMN "canReadEmployees" boolean DEFAULT true NOT NULL;
+ALTER TABLE "OrganizationEmployeeRole" ALTER COLUMN "canReadEmployees" DROP DEFAULT;
+--
+-- Add field canReadNewsItems to organizationemployeerole
+--
+ALTER TABLE "OrganizationEmployeeRole" ADD COLUMN "canReadNewsItems" boolean DEFAULT false NOT NULL;
+ALTER TABLE "OrganizationEmployeeRole" ALTER COLUMN "canReadNewsItems" DROP DEFAULT;
+--
+-- Add field canReadProperties to organizationemployeerole
+--
+ALTER TABLE "OrganizationEmployeeRole" ADD COLUMN "canReadProperties" boolean DEFAULT true NOT NULL;
+ALTER TABLE "OrganizationEmployeeRole" ALTER COLUMN "canReadProperties" DROP DEFAULT;
+--
+-- Add field canReadContacts to organizationemployeerolehistoryrecord
+--
+ALTER TABLE "OrganizationEmployeeRoleHistoryRecord" ADD COLUMN "canReadContacts" boolean NULL;
+--
+-- Add field canReadEmployeeRoles to organizationemployeerolehistoryrecord
+--
+ALTER TABLE "OrganizationEmployeeRoleHistoryRecord" ADD COLUMN "canReadEmployeeRoles" boolean NULL;
+--
+-- Add field canReadEmployees to organizationemployeerolehistoryrecord
+--
+ALTER TABLE "OrganizationEmployeeRoleHistoryRecord" ADD COLUMN "canReadEmployees" boolean NULL;
+--
+-- Add field canReadNewsItems to organizationemployeerolehistoryrecord
+--
+ALTER TABLE "OrganizationEmployeeRoleHistoryRecord" ADD COLUMN "canReadNewsItems" boolean NULL;
+--
+-- Add field canReadProperties to organizationemployeerolehistoryrecord
+--
+ALTER TABLE "OrganizationEmployeeRoleHistoryRecord" ADD COLUMN "canReadProperties" boolean NULL;
+
+--
+-- [CUSTOM] Set canReadNewsItems to true for existed administrators, managers and dispatchers
+--
+UPDATE "OrganizationEmployeeRole"
+SET "canReadNewsItems" = true
+WHERE name  = ANY(ARRAY['employee.role.Administrator.name', 'employee.role.Dispatcher.name', 'employee.role.Manager.name']);
```

Comment:
I've never seen syntax like this. IMO you can do that thing with something like that:
```suggestion
WHERE name  in ('employee.role.Administrator.name', 'employee.role.Dispatcher.name', 'employee.role.Manager.name');
```

Anyway if your variant works fine, you can ignore my suggest)

Reply:
for example `20211109074828-0078_extend_roles_20211111_0758.js` or `20220721114500-0152_manual_insert_common_contact_roles.js`